packages/run-protocol/src/econ-behaviors.js

```diff
@@ -442,3 +443,158 @@ export const startRewardDistributor = async ({
   );
 };
 harden(startRewardDistributor);
+
+/**
+ * TODO: refactor vats/core/types.js like this
+ *
+ * @template T
+ * @typedef {{
+ *   consume: { [P in keyof T]: ERef<T[P]> },
+ *   produce: { [P in keyof T]: Producer<T[P]> },
+ * }} PromiseMarket
+ */
+
+/**
+ * @typedef {EconomyBootstrapPowers & WellKnownSpaces & PromiseMarket<{
+ *   runStakeBundle: SourceBundle,
+ *   client: ClientManager,
+ *   lienBridge: StakingAuthority,
+ * }>} RunStakeBootstrapPowers
+ */
+
+/**
+ * @param {RunStakeBootstrapPowers } powers
+ * @param {Object} config
+ * @param {Rational} config.mintingRatio ratio of RUN minted to BLD
+ * @param {bigint} config.interestRateBP
+ * @param {bigint} config.loanFeeBP
+ * @param {bigint} config.chargingPeriod
+ * @param {bigint} config.recordingPeriod
+ * @typedef {[bigint, bigint]} Rational
+ * @typedef {Awaited<ReturnType<typeof import('./runStake/runStake.js').start>>} StartRunStake
+ */
+export const startRunStake = async (
+  {
+    consume: {
+      zoe,
+      // ISSUE: is there some reason Zoe shouldn't await this???
+      feeMintAccess: feeMintAccessP,
+      runStakeBundle,
+      lienBridge,
+      client,
+      chainTimerService,
+      economicCommitteeCreatorFacet,
+    },
+    // @ts-ignore TODO: add to BootstrapPowers
+    produce: { runStakeCreatorFacet, runStakeGovernorCreatorFacet },
+    installation: {
+      consume: { contractGovernor },
+      produce: { runStake: runStakeinstallR },
+    },
+    instance: {
+      consume: { economicCommittee },
+      produce: { runStake: runStakeinstanceR },
+    },
+    brand: {
+      consume: { BLD: bldBrandP, RUN: runBrandP },
+      produce: { Attestation: attestationBrandR },
+    },
+    issuer: {
+      consume: { BLD: bldIssuer },
+      produce: { Attestation: attestationIssuerR },
+    },
+  },
+  config = {
+    mintingRatio: [1n, 4n],
+    interestRateBP: 250n,
+    loanFeeBP: 200n,
+    chargingPeriod: SECONDS_PER_HOUR,
+    recordingPeriod: SECONDS_PER_DAY,
+  },
+) => {
+  const bundle = await runStakeBundle;
+  const [feeMintAccess, bldBrand, runBrand, governor, installation, timer] =
+    await Promise.all([
+      feeMintAccessP,
+      bldBrandP,
+      runBrandP,
+      contractGovernor,
+      E(zoe).install(bundle),
+      chainTimerService,
+    ]);
+
+  const installations = {
+    governor,
+    getRUN: installation,
+  };
+
+  const poserInvitationP = E(
+    economicCommitteeCreatorFacet,
+  ).getPoserInvitation();
+  const [initialPoserInvitation, electorateInvitationAmount] =
+    await Promise.all([
+      poserInvitationP,
+      E(E(zoe).getInvitationIssuer()).getAmountOf(poserInvitationP),
+    ]);
+
+  // t.log({ electorateCreatorFacet, electorateInstance });
```

Comment:
you too? I'm the only one who, since I needed it earlier, likes to keep it around in case I need it later?
My vague memory of working in the kernel says we used to keep them.

Oh. this one has no hope of working when it's un-commented. Never mind. gone.